### PR TITLE
jsonschema_validation: Group errors by type

### PR DESCRIPTION
**Pull request by ghost**
https://github.com/Open-Telecoms-Data/cove-ofds/issues/4 https://github.com/Open-Telecoms-Data/cove-ofds/issues/22

### CODE REVIEW

**Comment by ghost** [comment]
Would you 2 like to try this and see if it's ok (where ok == better than what's there!)

It isn't complete - it doesn't have all the right columns on the tables yet. But it does break up that panel by type and add the titles and description you wanted, and it improves the path.

**Comment by duncandewhurst** [comment]
Looks good! A few things to update (aside from the table columns):

* Add quotes to string values, for example, in the image below, 'yes' should be enclosed in double quotes because it is a string value. Otherwise, it would be impossible to distinguish between `true` and `"true"`.
* Render words in backticks as code, for example, in the image below, \`true\`, \`false\` and \`"\` should be rendered as `true`, `false` and `"` 
* Render markdown links as HTML links
image

**Comment by ghost** [comment]
Thanks!

> Add quotes to string values,

Moved to https://github.com/Open-Telecoms-Data/cove-ofds/issues/22#issuecomment-1309907233

> Render words in backticks as code

Moved to https://github.com/Open-Telecoms-Data/cove-ofds/issues/35

> Render markdown links as HTML links

Got 1, missed 2 - added second commit to fix those.

Over to you @Lathrisk

File: cove_ofds/templates/cove_ofds/jsonschema_validation_panel.html [hunk 1]
```diff
@@ -0,0 +1,151 @@
+{% load i18n %}
+
+{% if 'prefixItems' in validation_errors %}
+    <h4>{% trans 'prefixItems' %}</h4>
+    <p>DESCIRPTION</p>
```
Comment (Lathrisk) [inline cove_ofds/templates/cove_ofds/jsonschema_validation_panel.html:5]: Wee typo here (i.e. `DESCRIPTION` vs `DESCIRPTION`)

**Review by Lathrisk — APPROVED** [review]

File: cove_ofds/templates/cove_ofds/jsonschema_validation_panel.html (hunk 1 above)
Comment (ghost) [reply cove_ofds/templates/cove_ofds/jsonschema_validation_panel.html:5]: It's a placeholder anyway as the source spreadsheet was blank - I'll open a issue to point @duncandewhurst to it

**Pull request merged**